Nexogen.Libraries.Metrics.Prometheus.Standalone/PrometheusServer.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Nexogen.Libraries.Metrics.Prometheus;
+
+namespace Nexogen.Libraries.Metrics.Prometheus.Standalone
+{
+    /// <summary>
+    /// A simple Prometheus metric server using an embedded HTTP server.
+    /// </summary>
+    public class PrometheusServer : IHostedService
+    {
+        private readonly string _prefix;
```

Comment:
We don't use _ prefix in this project. Would you please remove those? Otherwise the PR looks fine, I will merge it and publish a pre-release Nuget after you have updated the PR.

Reply:
Updated the PR as requested.